client/src/components/Overview/subcomponents/SizeSelector.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import css from '../styles/sizeSelector.module.css';
+import Dropdown from './Dropdown.jsx';
+
+const SizeSelector = ({ currStyle, handleSizeSelect }) => {
+  const availableSkus = Object.keys(currStyle.skus)
+    .filter(sku => currStyle.skus[sku].quantity !== 0);
+  const availableSizes = {};
+  for (var sku of availableSkus) {
+    availableSizes[sku] = currStyle.skus[sku].size;
+  }
+
+  return (
+    <ul>
+      <Dropdown options={availableSizes} clickHandler={handleSizeSelect}/>
+      {/* {(isNoAvailble(currStyle)) ?
```

Comment:
Why is this commented?

Reply:
Kept if for further refactoring. Current drop down if half baked. Removed from this PR to keep our main code base clean. Thanks for pointing it out.